projects/storefrontstyles/scss/components/misc/popover/_popover.scss

```diff
@@ -0,0 +1,193 @@
+:root {
+  --cx-popover-font-size: 0.875rem;
+  --cx-popover-background: var(--cx-color-inverse);
+  --cx-popover-max-width: 280px;
+  --cx-popover-border-width: 1px;
+  --cx-popover-border-color: var(--cx-color-light);
+  --cx-popover-border-radius: 5px;
+  --cx-popover-arrow-width: 1rem;
+  --cx-popover-arrow-height: 0.5rem;
+  --cx-popover-arrow-color: var(--cx-popover-background);
+  --cx-popover-z-index: 10;
+  --cx-popover-box-shadow-color: rgba(0, 0, 0, 0.1);
+}
+
+%cx-popover {
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: var(--cx-popover-z-index);
+  display: block;
+  max-width: var(--cx-popover-max-width);
+  font-size: var(--cx-popover-font-size);
+  word-wrap: break-word;
+  background-color: var(--cx-popover-background);
+  background-clip: padding-box;
+  border: var(--cx-popover-border-width) solid var(--cx-popover-border-color);
+  border-radius: var(--cx-popover-border-radius);
+  padding: 0.5rem;
```

Comment:
padding should be: `padding:1rem 0.938rem;`

Reply:
Updated